mcs/class/System.Data/mobile_System.Data.dll.sources

```diff
@@ -64,6 +65,6 @@ System.Data.Common/DbTypes.cs
 System.Data.Common/ExceptionHelper.cs
 System.Data.Common/DbConnectionStringBuilderHelper.cs
 
-#include mobile_referencesource.sources
-
+MobileStubs.cs
```

Comment:
Where is this file? Also it should go to corefx/ folder if it's for corefx specific implementation